cirq-google/cirq_google/engine/processor_sampler_test.py

```diff
@@ -24,15 +24,29 @@
 @pytest.mark.parametrize('circuit', [cirq.Circuit(), cirq.FrozenCircuit()])
 def test_run_circuit(circuit):
     processor = mock.create_autospec(AbstractProcessor)
-    sampler = cg.ProcessorSampler(processor=processor)
+    run_name = "RUN_NAME"
```

Comment:
Add tests for the case where run_name and device_config_name are empty. Ditto for other `ProcessorSampler` methods.